docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -14,34 +14,99 @@ This how-to guide is still a work in progress and will change.
 * Clone of the https://github.com/appuio/terraform-openshift4-exoscale[terraform-openshift4-exoscale] repository
 * https://community.exoscale.com/documentation/dns/quick-start/#subscribing-to-the-service[DNS subscription] activated in the Exoscale organisation
+* `vault` https://www.vaultproject.io/docs/commands[Vault CLI]
+* `curl`
+* `gzip`
 * A copy of the Ursula Debian package
 
+[WARNING]
+====
+Make sure the version of openshift-install and the rhcos image is the same, otherwise ignition will fail.
+====
+
 == Cluster Installation
 
-. Register the new OpenShift 4 cluster in Lieutenant: https://control.vshn.net/syn/lieutenantclusters
+Register the new OpenShift 4 cluster in Lieutenant.
 
-. Configure input
+.Lietenant API endpoint
+****
+Use the following endpoint for Lieutenant:
+
+VSHN:: https://api.syn.vshn.net
+****
+
+=== Set up LDAP service
+
+. Create an LDAP service
 +
+Use https://control.vshn.net/vshn/services/_create to create a service.
+The name must contain the customer and the cluster name.
+And then put the LDAP service ID in the following variable:
++
+[source,console]
+----
+export LDAP_ID="Your_LDAP_ID_here"
+export LDAP_PASSWORD="Your_LDAP_pw_here"
+----
+
+=== Configure input
+
+.Access to various APIs
 [source,console]
 ----
 export EXOSCALE_ACCOUNT=<exoscale-account>
 export EXOSCALE_API_KEY=<exoscale-key>
 export EXOSCALE_API_SECRET=<exoscale-secret>
-export EXOSCALE_REGION=ch-dk-2
-export CLUSTER_ID=<cluster-name>
-export BASE_DOMAIN=<cluster-base-domain>
-export PULL_SECRET=<redhat-pull-secret> # From https://cloud.redhat.com/openshift/install/pull-secret
+export EXOSCALE_REGION=<exoscale-zone>
+
+# From https://git.vshn.net/profile/personal_access_tokens
+export GITLAB_TOKEN=<gitlab-api-token>
+
+# For example: https://api.syn.vshn.net
+# IMPORTANT: do NOT add a trailing `/`. Commands below will fail.
+export COMMODORE_API_URL=<lieutenant-api-endpoint>
+# See https://wiki.vshn.net/x/ngMBCg#ClusterRegistryinLieutenantSynthesizeaCluster(synfection)-Preparation
+export COMMODORE_API_TOKEN=<lieutenant-api-token>
+----
+
+.VSHN sepcific input
+[source,console]
+----
+export CLUSTER_ID=<lieutenant-cluster-id> # Looks like: c-<something>
+export TENANT_ID=$(curl -sH "Authorization: Bearer ${COMMODORE_API_TOKEN}" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r .tenant)
+----
 
+.OpenShift configuration
+[source,console]
+----
+export BASE_DOMAIN=<your-base-domain>
+export PULL_SECRET='<redhat-pull-secret>' # As copied from https://cloud.redhat.com/openshift/install/pull-secret "Copy pull secret". value must be inside quotes.
+----
+
+For `BASE_DOMAIN` explanation, see xref:explanations/dns_scheme.adoc[DNS Scheme].
+
+.Exoscale configuration
```

Comment:
I tried something with a recent commit. However, I'm not sure how we set the terraform variable ssh_key with vault. If we store the pubkey as a vault reference instead of plain text, would catalog compilation resp. `terraform apply` work in the output dir? Or do we expose a variable like the `lb_exoscale_api_secret` and set it with `TF_VAR_...` ?

Reply:
> If we store the pubkey as a vault reference instead of plain text, would catalog compilation resp. `terraform apply` work in the output dir? Or do we expose a variable like the `lb_exoscale_api_secret` and set it with `TF_VAR_...` ?

We don't need to have a Vault reference for the public key, we just need to save the private key in Vault, so it's available for debugging purposes.